src/utils/get-ticket-no.ts

```diff
@@ -3,21 +3,26 @@ export const IS_JIRA =
   window.location.pathname.startsWith('/browse/');
 
 export const getTicketNo = () => {
-  const urlParams = new URLSearchParams(window.location.search);
-  const azureParam = urlParams.get('workitem');
-
   const ticket = 'DEFAULT';
 
   if (IS_JIRA) {
     const jiraNo = window.location.href.match(/[A-Z]{2,}-\d+/);
 
     if (jiraNo !== null) return jiraNo[0];
   } else {
-    if (azureParam !== null) return `AZURE-${azureParam}`;
+    const urlParams = new URLSearchParams(window.location.search);
+    const azureParam = urlParams.get('workitem');
+
+    const project = window.location.href.match(
+      /azure.com\/(?<project>[A-Za-z]{2,})/
+    );
+
+    if (azureParam !== null) return `${project?.groups?.project}-${azureParam}`;
```

Comment:
we will have `undefined-123` if the above regex fails to match. minor issue if in case url is changed